demisto_sdk/commands/validate/tests/BA_validators_test.py

```diff
@@ -1018,3 +1021,54 @@ def test_IsValidVersionValidator_fix():
         == "Updated the content item version to -1."
     )
     assert content_item.version == -1
+
+
+ERROR_MSG_TEMPLATE = "The content item name is {} , which contains an excluded word."
+
+
+@pytest.mark.parametrize(
+    "content_items, expected_number_of_failures, expected_error_message",
+    [
+        # a correct integration display name
```

Comment:
let's use `pytest.param` with `id` to give the cases names

Reply:
Done